src/hooks/useApp.tsx

```diff
@@ -131,6 +132,18 @@ function useApp() {
     localStorage.setItem("confirmationState", JSON.stringify(confirmationState));
   }, [confirmationState]);
 
+  // Check for missed hint calls
+  useEffect(() => {
+    (async () => {
+      try {
+        await scheduledHintCalls();
```

Comment:
no need to use await here just apply the catch callback

Reply:
made the requested changes